### PR TITLE
refactor(DepartureCard): update time formats

### PR DESCRIPTION
**Pull request by thecristen**
#### Summary of changes

**Asana Ticket:** [Departure Card | Change Second Departure Time Format](https://app.asana.com/0/385363666817452/1205040060480939/f)

Consolidate `<DepartureList />` & `<DepartureTimes />` logic to render the list of departures, and implement the desired new formatting/styling for departure card times.

This removes a lot of JS-based logic around displaying times in the departure cards -- some no longer needed in the new design, the rest shifted into CSS. 

---

#### General checks
* [x] **Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?** This is a good practice that can facilitate easier reviews.
* [x] **Testing.** Do the changes include relevant passing updates to tests? This includes updating screenshots. Preferably tests are run locally to verify that there are no test failures created by these changes, before opening a PR.
* [x] **Tech debt.** Have you checked for tech debt you can address in the area you're working in? This can be a good time to address small issues, or create Asana tickets for larger issues.

### CODE REVIEW

File: apps/site/assets/ts/stop/__tests__/components/DepartureTimesTest.tsx [hunk 1]
```diff
@@ -339,299 +336,6 @@ describe("DepartureTimes", () => {
     });
   });
 
-  describe("infoToDisplayTime", () => {
```
Comment (i0sea) [inline apps/site/assets/ts/stop/__tests__/components/DepartureTimesTest.tsx:342]: If only the formatting is changing I think some of these tests are still valid

File: apps/site/assets/ts/stop/components/DepartureTimes.tsx [hunk 2]
```diff
@@ -39,69 +36,28 @@ const toInformativeAlertBadge = (alerts: Alert[]): JSX.Element | undefined => {
...
     // Informative badges need more padding between them and the time
-    alertClass = "pt-4";
+    alertClass += " pt-4";
   }
+
+  const timeList = departuresListFromInfos(
+    departures,
+    isCR,
+    targetDate,
+    isCR ? 1 : 2,
```
Comment (i0sea) [inline apps/site/assets/ts/stop/components/DepartureTimes.tsx:56]: question: is this a duplicate?

File: apps/site/assets/ts/stop/components/BasicTime.tsx [hunk 3]
```diff
@@ -20,8 +21,16 @@ const BasicTime = ({
       ? formatRelativeTime(time, targetDate)
       : formatToBostonTime(time, "h:mm aa");
 
+  // displayType === "relative" might produce an absolute-formatted time if it's
+  // more than an hour away. parse the result to find the truth.
+  const parsedTime = parse(formattedTime, "h:mm aa", new Date()).getTime();
+  const isAbsolute = Number.isNaN(parsedTime) ? "relative" : "absolute";
```
Comment (i0sea) [inline apps/site/assets/ts/stop/components/BasicTime.tsx:27]: question: what is an `absolute-formatted time`?

File: apps/site/assets/ts/stop/__tests__/components/DepartureTimesTest.tsx (hunk 1 above)
Comment (thecristen) [reply apps/site/assets/ts/stop/__tests__/components/DepartureTimesTest.tsx:342]: @i0sea Did you finish reading the PR? This function is deleted.

File: apps/site/assets/ts/stop/components/BasicTime.tsx (hunk 3 above)
Comment (thecristen) [reply apps/site/assets/ts/stop/components/BasicTime.tsx:27]: @i0sea 😅 Ryan had this same question on an earlier PR, maybe I need a better name for it.
It's the format that's not relative... absolute = 9:55AM, relative = 20 minutes

Comment (i0sea) [reply apps/site/assets/ts/stop/components/BasicTime.tsx:27]: maybe `standard format` and `minutes format` although thats a bit wordy 💭
